docs/modules/user-guide/partials/con_devfile.adoc

```diff
@@ -1,42 +1,53 @@
 [id="con_devfile_{context}"]
-= Devfile
+= About devfiles
 
 [role="_abstract"]
 
-Devfiles are YAML files that describe and define developer workspaces or cloud-native development environments. Devfiles also describe best practices for end-to-end application development and customize or extend the application development to fit a team's current development practices. Instead of forcing a new workflow or tools, devfiles provide instructions for automating manual processes.
+.What a devfile does for you
 
-Development tools consume devfiles and transform the definition into a cloud workspace composed of multiple containers and cloud-native resources. Also, users can create any number of identical workspaces from the same devfile. This way, different teams can use and share workspaces in various ways, including a devfile registry or together with the source code of a project.
+You can use devfiles to automate and simplify your development process. With devfiles, you record the instructions for configuring and running your software build environment as a YAML-formatted text file. Then, optionally, you can distribute those instructions to other software build tools and IDEs that support devfiles. Finally, to automatically configure and build a running application from a development project, you run the devfile instructions. You can also use devfiles to provide consistent software build processes across different languages and tools. 
```

Comment:
Consider to change "Then, optionally, you can distribute those instructions to other..." to "Then, optionally, you can distribute those devfiles to other..."

Reply:
We uses terms like "software build environment", "software build tools", "software build processes". Is the word "software" redundant? Isn't software is implicitly in the context of our docs? I am not sure if there is a reason for us to explicitly say software in this case for which we don't typically call that out on our docs.

Reply:
Thanks for the sentence change suggestion. And good catch, Elson, with the unneeded usage of "software." I've made the changes, thanks!